core/src/test/java/org/infinispan/metrics/ClusteredCacheManagerMetricsTest.java

```diff
@@ -77,17 +70,4 @@ public void testMetricsAreRegistered() {
       statsEviction.getId().getTags().contains(nodeNameTag);
       statsEviction.getId().getTags().contains(cacheManagerTag);
    }
-
-   private void verifyMeters(MeterRegistry registry) {
```

Comment:
Is the test failing with this check? I'm expecting the legacy to keep the same metrics as before.

Reply:
Don't fully remember. 😅 It may be because I found it superfluous. But I reverted it now.